controllers/drplacementcontrol.go

```diff
@@ -1048,6 +1048,9 @@ func (d *DRPCInstance) relocate(preferredCluster, preferredClusterNamespace stri
 	d.setDRState(drState)
```

Comment:
The comment above:

    // Make sure we record the state that we are failing over

Is copy paste from failover code?